front/app/components/Form/Components/Controls/LinearScaleControl.tsx

```diff
@@ -31,6 +33,7 @@ const LinearScaleControl = ({
 }: ControlProps) => {
   const isSmallerThanXlPhone = useBreakpoint('phone');
   const maximum = schema?.maximum;
+  const answerNotPublic = uischema.options?.answer_visible_to === 'admins';
```

Comment:
I don't know exactly under which conditions `uischema.options` can be `undefined`, but with the current implementation `answerNotPublic` will be `false` (so the answer will be public) if `uischema.options` is `undefined`. Just making sure that's intended behavior, don't know the context

Reply:
Yes for now at least that's the intended behavior :) In the future we want `answer_visible_to` to be configurable for each field in surveys/ideation, but we don't have that yet. Thanks for bringing it up!! :+1: